isis/src/base/apps/caminfo/caminfo.xml

```diff
@@ -39,164 +39,212 @@
 Object = Caminfo
   Object = Parameters
     Program     = caminfo
-    IsisVersion = "3.7.0  | 2019-04-16"
-    RunDate     = 2019-04-24T23:29:22
-    IsisId      = MeSSEnGeR/MDIS-WAC/1/0211286081:889000
-    From        = EW0211286081G.lev1.cub
-    Lines       = 1024
-    Samples     = 1024
+    IsisVersion = "6.0.0  | 2021-08-10"
+    RunDate     = 2021-08-18T14:53:40
+    IsisId      = Viking1/VISB/33322515
+    From        = default.cub
+    Lines       = 1056
+    Samples     = 1204
     Bands       = 1
   End_Object
 
   Object = Camstats
-    MinimumLatitude           = 46.17349725649
-    MaximumLatitude           = 54.072584907617
-    MinimumLongitude          = 284.17017788557
-    MaximumLongitude          = 296.47498126014
-    MinimumResolution         = 235.10973750118
-    MaximumResolution         = 238.32543228025
-    MinimumPhase              = 50.107674499885
-    MaximumPhase              = 60.748154060087
-    MinimumEmission           = 0.0076793870352429
-    MaximumEmission           = 11.494912087698
-    MinimumIncidence          = 52.486663651971
-    MaximumIncidence          = 58.235714475896
-    LocalTimeMinimum          = 9.7555337547863
-    LocalTimeMaximum          = 10.575853979758
-    ObliqueResolutionMinimum = 235.23505597394
-    ObliqueResolutionMaximum  = 243.18035398447
+    LatitudeMinimum                          = 9.9286479874788
+    LatitudeMaximum                          = 10.434709753119
+    LatitudeAverage                          = 10.181983206084
+    LatitudeStandardDeviation                = 0.11084102743244
+    LongitudeMinimum                         = 255.64554871862
+    LongitudeMaximum                         = 256.14606952525
+    LongitudeAverage                         = 255.89390491018
+    LongitudeStandardDeviation               = 0.10658330458136
```

Comment:
If we were trying to avoid breaking backwards compatibility won't some of these being `(something)Minimum` rather than `Minimum(something)` break that?

Reply:
I would agree here - even if its in the documentation as such. The output PVL currently has these keywords at the very top of the Camstats Group, but it appears the new logic prints these out in the order they occur in the table. Should the existing/current order be retained as well?

Reply:
It looks as tho the Minimum/Maximum keywords are not included at all in the XML doc file.

Reply:
Ah I missed updating the XML example I'll get that done. The code prints out the values as both new and old.

I would not take order as part of backward compatibility, both PVL and CSV with headers account for different orders by allowing access by keyword and column name respectively.